Controller/MediaAdminController.php

```diff
@@ -77,10 +77,12 @@ public function listAction(Request $request = null)
         $datagrid->setValue('context', null, $context);
 
         // retrieve the main category for the tree view
-        $category = $this->container->get('sonata.classification.manager.category')->getRootCategory($context);
+        $rootCategory = $this->container->get('sonata.classification.manager.category')->getRootCategory($context);
+        // This should be safe as the root category has to exist for a given context but I do not like fatal errors
```

Comment:
I agree but seriously, it took me 1 hour to understand and fix the code (I never read before this part of the sonata bundles) and it takes me more to fit the "administrative part" of the process.

The code is fixed and the original behaviour is preserved. I think this is sufficient for maintenance release. Maybe someone else could do the other part ? It would be great, thank you.